openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -168,18 +164,55 @@ def add_to_transaction(neo4j_entities, transaction):
             transaction.create(entity)
 
     @staticmethod
-    def should_dump_course(course_key):
+    def get_command_last_run(course_key, graph):
+        """
+        This information is stored on the course node of a course in neo4j
+        :param course_key: a CourseKey
+        :param graph: a py2neo Graph
+        :return: a string representation of when this course was last dumped to
+         neo4j or None
+        """
+        selector = NodeSelector(graph)
+        course_node = selector.select(
```

Comment:
What can be possibilities for the datatype of  `course_node`?  `dict` and `None`?